scripts/test.sh

```diff
@@ -11,7 +11,7 @@ for f in $( git diff-tree $TRAVIS_COMMIT --name-status -r | grep php | grep -v "
 
 # Run the unit tests
 # phpunit
-phpunit
+vendor/bin/phpunit --debug
```

Comment:
Yep. Otherwise it doesn't print the failing test. I tried to find a better test reporter than `dot` but my phpunit-fu was inadequate.